index.html

```diff
@@ -76,9 +76,9 @@
             <li><a href="#signup">Sign Up</a></li>
             <li><a href="#tickets">Tickets</a></li>
             <li><a href="#cfp">Call For Proposals</a></li>
+            <li><a href="#geektrain">Geek Train</a></li>
```

Comment:
Looking good, but do we want this to be a top-level navigation item? Maybe instead we could make a "schedule" item here, and start building that out, with the GeekTrain as the first piece of that? Also, schedule should probably be a separate page, cause it's going to get large.